src/main/webapp/cache-status/cache-status.html

```diff
@@ -109,7 +109,33 @@ <h2 class="card-pf-title">
 
     <div class="row row-cards-pf">
       <!-- BEGIN CARD -->
-      <div class="col-xs-2 col-sm-2 col-md-2" match-height="ispn-card">
+      <div class="col-xs-6 col-sm-4 col-md-3" match-height="ispn-card"
+           ng-if="currentCacheStats.firstServerStats['offline-sites'].length > 0
+               || currentCacheStats.firstServerStats['online-sites'].length > 0
+               || currentCacheStats.firstServerStats['mixed-sites'].length > 0 ">
+        <div class="card-pf card-pf-accented ispn-card">
+          <div class="card-pf-heading">
+            <h2 class="card-pf-title">Backup sites</h2>
+          </div>
+          <div class="card-pf-body">
+            <div class="row">
+              <div class="col-md-12">
+                <table class="col-sm-12 col-xs-12 col-md-12">
+                  <tbody>
+                  <tr ng-repeat="site in currentCacheStats.firstServerStats['offline-sites']"><th><span class="pficon pficon-error-circle-o"></span> <span class="label label-danger">{{site}}</span></th></tr>
```

Comment:
You can also encapsulate currentCacheStats.firstServerStats['offline-sites'] and the likes under some function in scope. Perhaps it can be reused and we do not leak as much details from the controller into view. I mean, I am also learning about these details myself :-)